pkgs/development/python-modules/rectangle-packer/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  setuptools,
+  cython,
+}:
+
+buildPythonPackage rec {
+  pname = "rectangle-packer";
+  version = "2.0.2";
```

Comment:
```suggestion
  version = "2.0.2";
  pyproject = true;
```

Reply:
The project doesn't include a pyproject file. is this useful anyway?